general/CatalogoDatiInformativiPSP.xsd

```diff
@@ -1,550 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@andreapasuch @corvettafabio we should verify this delete

Reply:
PR non approvabile. Questo file non può essere eliminato